exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,8 @@
-// Declare your function here
-
+// Create a greeting message by name and age
+function createLongGreeting(name, age) {
+  var message = `Hello, my name is ${name} and I'm ${age} years old`;
```

Comment:
I belive for this particular function, there is no need to declare the variable `message` as it is only used inside the function, so you can write the code as `return `Hello, my name is ${name} and I'm ${age} years old`;`